### DISCUSSION TITLE
Updating the play count of songs

### DISCUSSION THREAD

**Discussion by Amatsu-Kami** [jeffvli/feishin#328] [Q&A] [answered]
Hey,
the play counts of songs doesn't seems to be updating with navidrome, is it normal ?

**Comment by jeffvli** [comment]
If not enabled already, make sure the `Scrobble` setting is enabled under playback. That's what sends plays to your server.

**Reply by Amatsu-Kami** [reply]
I don't see the setting `Scrobble` in playback. I'm running feishin in a docker container, maybe that's why ?

**Reply by jeffvli** [reply, marked as answer]
Ah yep, that's a bug on my side... I'll unhide the scrobble settings for the web player in the next release.

**Reply by Amatsu-Kami** [reply]
Great, thanks ! 😊
